data/gegno/gegno intro missions.txt

```diff
@@ -28,7 +28,7 @@ mission "First Contact: Gegno Vi"
 	clearance
 	passengers 1
 	to offer
-		not "Gegno Genocide Defense: offererd"
+		not "Gegno Genocide Defense: offered"
```

Comment:
lol this is actually a big bugfix, @Saugia just pinging you so you can merge this when you have the time